src/astra_driver.cpp

```diff
@@ -165,6 +165,7 @@ AstraDriver::AstraDriver(ros::NodeHandle& n, ros::NodeHandle& pnh) :
   diagnostics_updater_.verbose_ = false;
   diagnostics_updater_.setHardwareID("astra_" + serial_number_);
   diagnostics_updater_.add("camera_availability", this, &AstraDriver::getAvailabilityDiagnostic);
+  diagnostics_updater_.add("depth_grabability", this, &AstraDriver::getDepthGrabability);
```

Comment:
I dont think grabability is the right word here.  No body grabs the image. maybe just depth_availability?